content/docs/manuals/unikernel-operations.md

```diff
@@ -1,12 +1,134 @@
 ---
-title: "Unikernel Operations"
+title: "Unikernel Deployment"
 summary: ""
 draft: false
 weight: 301
 toc: true
 seo:
-  title: "" # custom title (optional)
-  description: "" # custom description (recommended)
+  title: "Unikernel Deployment" # custom title (optional)
+  description: "Deployment of Unikraft Unikernels in Oakestra" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
   noindex: false # false (default) or true
----
\ No newline at end of file
+---
+
+Oakestra supports the deployment of unikernels built using [Unikraft](http://unikraft.org). 
+
+#### What do you need?
+It's simple. All you need to start deploying unikernels is:
+
+1. A unikernel .tar.gz hosted somewhere accessible to the Oakestra.
+2. A service deployment descriptor 
+3. A worker Node with KVM Runtime enabled
+
+## Packaging your first Unikraft image 
+
+Once you build your first Unikernel using [Unikraft](http://unikraft.org), you can create a tarball of the kernel image and the necessary files.
+
+```bash
+myunikernel.tar.gz
+|-- kernel
+`--files1/
+      |--...
+ `--...
+```
+
+Your unikernel tarball MUST contain a file named `kernel`, which is the unikernel image itself. Additionally, it CAN contain a `files1` folder, which is mounted by Oakestra as kernel filesystem at runtime. This folder contains all the additional files for the unikernel to run.
+
+{{< callout context="tip" title="Looking for an example?" icon="outline/rocket" >}}
+Check out the **Unikraft Nginx** example in the Application Catalogue section of the wiki.
```

Comment:
```suggestion
Check out the **[Unikraft Nginx](../app-catalog/nginx-unikernel-deployment/)** example in the Application Catalogue section of the wiki.
```
This is definitely requires a link, but it won't work until this branch is rebased onto the app-catalog branch

Reply:
created an issue to fix this when that's merged. Broken links are harder to spot than missing ones :D